docs/wiki/Upgrade from 0.3.1 to 0.4.0.md

```diff
@@ -2,7 +2,7 @@ In this area, we provide instructions to migrate from CARML version 0.3.1 to ver
 
 # Modules
 ## General import of new modules
-New modules can usually be migrated by adding the content of the release's `arm` folder into your own `arm` folder.
+New modules can usually be migrated by adding the content of the release's `modules` folder into your own `modules` folder.
```

Comment:
Undo any changes to this file. This release will remain with the 'ARM' folder

Reply:
Good point! Restoring the file.